content/organisation.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Organisation"
+date: 2022-02-01T11:53:42Z
+draft: false
+---
+<center> {{< figure src="/images/swifthep-organisation.png" alt="swift-hep Organisation" width="600px">}} </center>
```

Comment:
Since this is markdown, I would avoid HTML as much as possible.
If something specific is needed, Hugo has `shortcodes` that can be defined for that purpose.